streamlit_demo/app.py

```diff
@@ -0,0 +1,36 @@
+import tempfile
+
+import numpy as np
+import pysinsy
+import soundfile as sf
+import streamlit as st
+from nnmnkwii.io import hts
+from nnsvs.pretrained import create_svs_engine
+
+st.title("NNSVS Demo")
+st.markdown("Upload your .xml music file with text as input to make it sing.")
+
+models = {
+    "kiritan": "r9y9/20220321_kiritan_timelag_mdn_duration_mdn_acoustic_resf0conv",
+    "yoko": "r9y9/20220322_yoko_timelag_mdn_duration_mdn_acoustic_resf0conv",
+}
+
+voice_option = st.selectbox("Select the voice", models.keys())
+uploaded_file = st.file_uploader("Choose a .xml music file", type="xml")
+
+if st.button("synthesis") and uploaded_file:
+    with st.spinner(f"Synthesizing to wav"):
```

Comment:
```suggestion
    with st.spinner("Synthesizing to wav"):
```

Reply:
This is causing CI failures with pysen. Once this is fixed, CI will be all green. Then I will merge this PR